cmd/gateway/zerostor/api.go

```diff
@@ -0,0 +1,319 @@
+package zerostor
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"sync"
+
+	"github.com/google/uuid"
+	"github.com/gorilla/mux"
+	"github.com/minio/minio/cmd/gateway/zerostor/repair"
+	log "github.com/sirupsen/logrus"
+	"github.com/threefoldtech/0-stor/client/datastor/pipeline/storage"
+)
+
+var (
+	newLine = []byte{'\n'}
+)
+
+// HealerAPI type
+type HealerAPI struct {
+	listen string
+	cfg    ConfigManager
+	jobs   map[string]HealJob
+	m      sync.RWMutex
+}
+
+// NewHealerAPI creates a new healer api that listens on thie address
+func NewHealerAPI(listen string, cfg ConfigManager) *HealerAPI {
```

Comment:
I would be cool that the healerAPI also populate information to the prometheus /metrics handler.
Things like the info of healing jobs

Reply:
I don't see how this would work. The job results (like the summary) are not time based. Hence i don't see how this going to be meaningful in Prometheus

Reply:
I'm not talking about the summary, but more the jobs themself.
A metric to know the amount of jobs currently running...